app/src/contexts/observationsTableContext.tsx

```diff
@@ -574,11 +550,16 @@ export const ObservationsTableContextProvider = (props: PropsWithChildren<Record
   }, [_muiDataGridApiRef, addedRowIds, rows]);
 
   const refreshObservationRecords = useCallback(async () => {
+    const sort = firstOrNull(sortModel);
     return observationsContext.observationsDataLoader.refresh({
-      page: paginationModel.page,
-      limit: paginationModel.pageSize
+      limit: paginationModel.pageSize,
+      sort: sort?.field || undefined,
+      order: sort?.sort || undefined,
+
+      // API pagination pages begin at 1, but MUI DataGrid pagination begins at 0.
```

Comment:
I notice that in the postgres queries, pagination is 0 based. Is there any reason we convert from 0 to 1 on the frontend, and then back to 0 on the backend (rather than just have it 0 based the entire time)?

Reply:
The motivation to have 1-indexed pagination was because exposing the first page as `page=1` to the user seemed more sensible than `page=0`. Although the page number isn't really "exposed" to the user outside of the page number we show in the frontend, I think at the time this was decided, having the pagination query persist in the browser URL was still up in the air. So it made sense that the first page should naturally be `1`. Also, the frontend and backend code were developed independantly, so the zero-to-one and one-to-zero conversions were made on both sides to satisfy this decision. Maybe if we had picked a word like `offset` to go in the query parameters... I think `offset=0` would be totally acceptable. I still lean towards page number begins at `1` but I'm open to opposing arguments 😅

Reply:
Gotcha. Not a big deal either way, really. The UI could render `page+1` to the user, even if the URL tracked `page=0` (I don't think the average user is going to notice or care if the URL says page 0). I think making the rest of the code simpler is what I would prioritize. But, I really don't think its necessary to change anything at this point. As long as everything is well documented/typed, then it really shouldn't matter one way or the other. I was just curious if there was some other reason for avoiding 0 that I wasn't aware of.